src/tests/banners/EnvironmentMomentBannerTest.ts

```diff
@@ -0,0 +1,37 @@
+import { BannerPageTracking, BannerTargeting, BannerTest } from '../../types/BannerTypes';
+
+export const EnvironmentMomentBannerPath = 'environment-moment-banner.js';
+
+export const EnvironmentMomentBannerSupporters: BannerTest = {
+    name: 'EnvironmentMomentBannerSupporters',
+    bannerChannel: 'contributions',
+    testAudience: 'AllExistingSupporters',
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    canRun: (targeting: BannerTargeting, pageTracking: BannerPageTracking) => true,
```

Comment:
Let's have all 3 `canRun`s returning false initially, so that we can test in PROD with the ad takeover (using `force-banner`).
Ideally we could control this with a swich in the admin-console, but for now how about a hardcoded `const` somewhere?

Reply:
yeah makes sense! I've just chucked it in one of the test files - idk if there's a more appropriate spot for it?

Reply:
👍 that's fine, it's temporary!